manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml

```diff
@@ -80,6 +80,16 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
+        - name: liveness-probe
+          image: quay.io/k8scsi/livenessprobe:v1.1.0
```

Comment:
latest version seems to be 2.1.0, why we are using older version?

Reply:
a typo I think ...